src/test/java/org/simulator/stochastic/StochasticTestSuiteTest.java

```
@@ -155,7 +298,7 @@ public void testModel() throws IOException {
         // Runs the stochastic simulation
         boolean errorInSimulation = false;
         try {
-          runSimulation(sim, obs, orderedArgs);
+          sim.start((Double)orderedArgs.get(TIME));
         } catch (Exception e) {
           errorInSimulation = true;
```

Please avoid hiding the actual exceptions.